src/components/Quote.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+
+const Quote = () => {
+  const [quote, setQuote] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
- [ ] [OPTIONAL] I think the initial state for loading should be false, as when we start data fetching we set it to true :+1: